recipes/hipace/bld.bat

```diff
@@ -0,0 +1,45 @@
+@echo on
+
+:: simple install prep
+::   copy all hipace*.exe and hipace*.dll files
+if not exist %LIBRARY_PREFIX%\bin md %LIBRARY_PREFIX%\bin
+if errorlevel 1 exit 1
+
+:: configure
+cmake ^
```

Comment:
Please add the BUILD_SHARED_LIBS CMake variable. We do not allow shipping static libraries.

Reply:
The static library being shipped is AMReX, which is not this package. Definitely, you should not be shipping the headers / libraries of another package with this package. If you need to link your binary to AMReX statically because you are using a non-release / patched version that's probably fine, but if you are linking against one of their [stable releases](https://github.com/AMReX-Codes/amrex/releases), we would like you to package AMReX separately and link to it dynamically.

Reply:
Thx. Disabling the lib via `AMReX_INSTALL=OFF`

Reply:
Update: now a clean
```
Install the project...
$BUILD_PREFIX/bin/cmake -P cmake_install.cmake
-- Install configuration: "Release"
-- Installing: $PREFIX/bin/hipace.NOMPI.NOACC.DP
```

Thanks for catching this!